src/pages/Notifications/List/BundlePage.tsx

```diff
@@ -26,22 +28,26 @@ export const NotificationListBundlePage: React.FunctionComponent<NotificationLis
     const eventLogPageUrl = React.useMemo(() => linkTo.eventLog(props.bundle.name), [ props.bundle.name ]);
 
     return (
-        <>
-            <PageHeader
-                title={ `${Messages.pages.notifications.list.title} | ${props.bundle.displayName}` }
-                subtitle={ <span>This service allows you to configure which notifications different
-                    users within your organization will be entitled to receiving. To do this, create behavior groups and apply
-                    them to different events. Users will be able to opt-in or out of receiving authorized event notifications in their
-                <a href={ localUrl(`/user-preferences/notifications/${props.bundle.name}`,
-                    getInsights().chrome.isBeta()) }> User Preferences</a>.</span> }
-                action={ <ButtonLink isDisabled={ !rbac.canReadEvents } to={ eventLogPageUrl } variant={ ButtonVariant.secondary }>
-                    { Messages.pages.notifications.list.viewHistory }
-                </ButtonLink> }
-            />
-            <Main>
+        <><PageHeader
+            title={ `${Messages.pages.notifications.list.title} | ${props.bundle.displayName}` }
+            subtitle={ <span>This service allows you to configure which notifications different
+                users within your organization will be entitled to receiving. To do this, create behavior groups and apply
+                them to different events. Users will be able to opt-in or out of receiving authorized event notifications in their
+            <a href={ localUrl(`/user-preferences/notifications/${props.bundle.name}`,
+                getInsights().chrome.isBeta()) }> User Preferences</a>.</span> }
+            action={ <ButtonLink isDisabled={ !rbac.canReadEvents } to={ eventLogPageUrl } variant={ ButtonVariant.secondary }>
+                {Messages.pages.notifications.list.viewHistory}
+            </ButtonLink> } />
+
+        <TabComponent elements={ [
+            <Main key='0'>
                 <SplunkBetaEnvironmentBanner />
                 <BundlePageBehaviorGroupContent applications={ props.applications } bundle={ props.bundle } />
+            </Main>,
+            <Main key='1'>
+                <TimeConfigComponent />
             </Main>
+        ] } />
```

Comment:
Step 2 (will require fix imports)

```suggestion
        <TabComponent>
            <Tab eventKey={ 0 } title={ <TabTitleText>Configuration</TabTitleText> }>
                 <Main>
                     <SplunkBetaEnvironmentBanner />
                     <BundlePageBehaviorGroupContent applications={ props.applications } bundle={ props.bundle } />
                 </Main>
            </Tab>
            <Tab eventKey={ 1 } title={ <TabTitleText>Settings</TabTitleText> }>
                <Main>
                    <TimeConfigComponent />
                </Main>
            </Tab>
        </TabComponent>
```